addons/mail/static/src/core/web/chatter.xml

```diff
@@ -51,8 +50,8 @@
                             <span class="d-flex invisible text-nowrap">
                                 <i class="me-1 fa fa-fw fa-eye-slash"/><t t-out="followingText.length > unfollowText.length ? followingText : unfollowText"/>
                             </span>
-                            <span class="position-absolute end-0 top-0" t-att-class="{ 'text-warning': unfollowHover.isHover }">
-                                <i class="me-1" t-att-class="{ 'fa fa-fw fa-times': unfollowHover.isHover, 'fa fa-fw fa-check': !unfollowHover.isHover }"/><span t-out="unfollowHover.isHover ? unfollowText : followingText"/>
+                            <span class="o-mail-Chatter-follow-container position-absolute end-0 top-0" t-att-class="{ 'text-warning': unfollowHover.isHover }">
+                                <span t-att-class="{ 'o-mail-Chatter-unfollow-text' : unfollowHover.isHover, 'o-mail-Chatter-follow-text' : !unfollowHover.isHover }" t-out="unfollowHover.isHover ? unfollowText : followingText"/>
```

Comment:
There's no reason to keep the inner `span`, we could just `t-out` on the span above.

And we could rename to something simpler like `o-mail-Chatter-following` (or `o-mail-Chatter-follow`, if the already used one is slightly changed to put a classname as modifier, e.g. `o-follow`). That way the classnames in tests stay small and easy to grasp at a glance.